src/client/components/UserPage/AnnouncementModal/index.tsx

```diff
@@ -229,6 +230,7 @@ export default function AnnouncementModal() {
             size="large"
             variant="text"
             className={classes.learnMoreButton}
+            onClick={()=> {GAEvent('Announcement Page', 'redirected to GoDirectory')}}
```

Comment:
Why the hardcoded reference to GoDirectory? the announcement modal is meant for more than that

Reply:
In the future when we change the subject of the announcement modal, we will change the event sent as well. So having the reference to godirectory will let us know in the google analytics dashboard that those existing clicks were referring to godirectory

Reply:
but do you really want to change the codebase every time an announcement has been made?

Reply:
i fear that this might defeat the purpose of allowing announcemodal title/message/descriptions to be configurable by env vars in the first place. If we need to update code to change announcement modal behaviour, then we might as well define every aspect of the modal in code